src/services/genre_detector.py

```
@@ -2,36 +2,53 @@
 import logging
 from typing import Set
 
+
 class GenreDetector:
     def __init__(self):
-        self.genres: Set[str] = {'News', 'Technology', 'Science', 'Entertainment', 'Sports', 'Other'}
+        self.genres: Set[str] = {
```

Ruff (0.8.0)

Use `set` instead of `Set` for type annotation

Replace with `set`

(UP006)